src/v2/wallet.js

```diff
@@ -1135,6 +1136,29 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
   }).call(this).asCallback(callback);
 };
 
+Wallet.prototype.accelerateTransaction = function(params, callback) {
+  return co(function *() {
+    // TODO(BG-9349): change the last check to > 0 and the error message once platform allows multiple transactions to
+    //                be bumped in the same CPFP transaction
+    if (_.isUndefined(params.cpfpTxIds) || !Array.isArray(params.cpfpTxIds) || params.cpfpTxIds.length !== 1) {
+      const error = new Error('expecting cpfpTxIds to be an array of length 1');
+      error.code = 'cpfptxids_not_array';
+      throw error;
+    }
+
+    if (_.isUndefined(params.cpfpFeeRate) && _.isUndefined(params.maxFeeRate)) {
+      debug('neither cpfpFeeRate nor maxFeeRate set in accelerateTransaction');
+    }
+
+    if (_.isUndefined(params.maxFee)) {
```

Comment:
Specifying the maximum absolute fee is of limited use here I think since we don't yet know the size of the child transaction, or how many parent bytes it needs to pay fees for.

I think a more sensible approach might be to enforce the client constants max fee rate, unless an override flag is passed (like the `ignoreMaxFeeRate` flag in v1). We should try to be safe by default, but allow callers to opt-in to potentially unsafe behavior if they explicitly choose to do so.

Reply:
@tylerlevine: The problem is that the customer may not know how large the corresponding ancestor set is that will be targeted by the child-pays-for-parent transaction. A flat `maxFee` will give the customer a safety net, so that they don't spend up spending something insane like more than 0.05BTC.

Reply:
The problem with a default `maxFeeRate` is also the fact that CPFP transactions will have abnormally large fee rates by design. I would imagine a non-trivial amount of CPFP transactions would be over this `maxFeeRate` and thus need an `ignoreMaxFeeRate` flag. We've made it clear in the documentation that setting a `maxFee` for CPFP transactions is recommended. However, I would like to make the debug line from this if statement more noisy, any ideas in that regard?

Reply:
Maybe this behavior makes sense:
1) If the CPFP tx fee rate is less than the client constants max fee rate, we don't print anything
2) If the CPFP tx fee rate exceeds the client constants max fee rate, we print a debug log which outputs both the client constants max fee rate, and the fee rate which is actually paid by the tx
3) If the CPFP tx fee rate exceeds twice the client constants max fee rate, we throw an error, unless maxFee is also passed and the total fee paid by the CPFP tx is less than the provided maxFee

I'm open to other suggestions on this as well.

Reply:
After some more discussion, I think we are good if we just enforce a sensible default maxFee on the platform side, which can be optionally overridden by passing an explicit `params.maxFee`.

Reply:
As of right now there is no default in the platform diff for `maxFee` and frankly, I am not sure what a sensible default would be. Instead, I enforce that `maxFee` is passed, unless explicitly not passed via the `noMaxFee` param. I do the same thing for `cpfpFeeRate`.

Reply:
I think that sounds fine for the time being as long as the enforcement of `maxFee` only affects cpfp.